inception/inception-search-mtas/src/test/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasDocumentIndexTest.java

```diff
@@ -456,6 +536,192 @@ public void testAnnotationQuery() throws Exception
+
+        StatisticsResult queryStatsResults = searchService.getQueryStatistics(user, project, query,
+                minTokenPerDoc, maxTokenPerDoc, features);
+
+        assertThat(queryStatsResults.getMinTokenPerDoc()).isEqualTo(minTokenPerDoc);
+        assertThat(queryStatsResults.getMaxTokenPerDoc()).isEqualTo(maxTokenPerDoc);
+        assertThat(queryStatsResults.getUser()).isEqualTo(user);
+        assertThat(queryStatsResults.getProject()).isEqualTo(project);
+
+        Map<String, LayerStatistics> expected = new HashMap<String, LayerStatistics>();
+
+        LayerStatistics expectedSearch = new LayerStatistics(1, 1, 0, 0.5, 0.5, Math.pow(0.5, 0.5),
+                0.5, 0.0, 0.25, 0.25, Math.pow(0.125, 0.5), 2);
+
+        expected.put("query.moon", expectedSearch);
+
+        assertTrue(expectedResults.equals(statsResults.getNonNullResults()));
+        assertTrue(expected.equals(queryStatsResults.getNonNullResults()));
+
+    }
+
+    private static CAS loadCas(String aPathToXmi) throws Exception
+    {
+        try (FileInputStream fis = new FileInputStream(getResource(aPathToXmi))) {
+            JCas jcas = JCasFactory.createJCas();
+            CasIOUtils.load(fis, jcas.getCas());
+            return jcas.getCas();
+        }
+    }
+
+    public static File getResource(String aResourceName)
+    {
+        return Paths.get(aResourceName).toFile();
+    }
+
+    public void uploadLargeProject(String aFolderPath, Project aProject, User aUser)
+        throws Exception
+    {
+        try (CasStorageSession casStorageSession = CasStorageSession.open()) {
+            for (final File fileEntry : getResource(aFolderPath).listFiles()) {
+
+                String xmiPath = aFolderPath + "\\" + fileEntry.getName();
+
+                CAS cas = loadCas(xmiPath);
+
+                SourceDocument sourceDocument = new SourceDocument();
+
+                sourceDocument.setName(fileEntry.getName());
+                sourceDocument.setProject(aProject);
+                sourceDocument.setFormat("text");
+
+                String sourceContent = cas.getDocumentText();
+
+                try (InputStream fileStream = new ByteArrayInputStream(
+                        sourceContent.getBytes(UTF_8))) {
+                    documentService.uploadSourceDocument(fileStream, sourceDocument);
+                }
+
+                documentService.writeAnnotationCas(cas,
+                        documentService.createOrGetAnnotationDocument(sourceDocument, aUser),
+                        false);
+            }
+        }
+        await("Waiting for indexing process to complete") //
+                .atMost(60, SECONDS) //
+                .pollInterval(5, SECONDS) //
+                .until(() -> searchService.isIndexValid(aProject)
+                        && searchService.getIndexProgress(aProject).isEmpty());
+    }
+
+    // executing this method takes around 30 minutes!
+    @Test
+    public void testStatisticsWithRealProject() throws Exception
```

Comment:
Please add an @Ignore here or better delete the test, because noone except you has the data. You could also instead use data from the internet,see e.g. for recommender tests that load it via DKPro